pixi.toml

```diff
@@ -83,13 +83,12 @@ gsw = "*"
 
 [feature.docs.dependencies]
 numpydoc = "!=1.9.0"
-nbsphinx = "*"
+myst-nb = "*"
 ipython = "*"
 sphinx = "*"
 pandoc = "*"
 pydata-sphinx-theme = "*"
 sphinx-autobuild = "*"
-myst-parser = "*"
 sphinxcontrib-mermaid = "*"
```

Comment:
More context around why this was removed?

Reply:
To avoid conflict, as it is already installed by `myst-nb`: https://myst-nb.readthedocs.io/en/latest/quickstart.html#get-started